blueocean-rest/src/main/java/io/jenkins/blueocean/analytics/Analytics.java

```diff
@@ -62,5 +68,49 @@ public static boolean isAnalyticsEnabled() {
     /**
      * @param req to track
      */
-    public abstract void track(TrackRequest req);
+    public final void track(TrackRequest req) {
```

Comment:
This probably should be in implementation. Keeping it fairly abstract, like how it was is better, giving implementations chance to track differently if they like.